lib/LANraragi.pm

```diff
@@ -194,7 +195,7 @@ sub startup {
     start_shinobu($self);
 
     # Check if this is a first-time installation.
-    LANraragi::Model::Config::first_install_actions();
+    LANraragi::Model::Setup::first_install_actions();
```

Comment:
If you're doing a use qw, you shouldn't need to specify the namespace when calling `first_install_actions`.